src/Bottleneck_distance/doc/Intro_bottleneck_distance.h

```diff
@@ -44,6 +47,43 @@ namespace persistence_diagram {
  * This implementation is based on ideas from "Geometry Helps in Bottleneck Matching and Related Problems"
  * \cite DBLP:journals/algorithmica/EfratIK01. Another relevant publication, although it was not used is
  * "Geometry Helps to Compare Persistence Diagrams" \cite Kerber:2017:GHC:3047249.3064175.
+ * 
+ * \section bottleneckdistanceprecision Distance computation
+ *
+ * Bottleneck distance does not use Euclidean distance, like explained in the following example:
```

Comment:
Looking at the whole page, this seems too prominent. It is just supposed to be a remark, but it appears as the first sentence of the section on computation...

Reply:
I tried to fix it on 4b401e9 
The figure was also incorrect ((13,0) instead of (0,13))
Tell me if the figure and the distance computation example is "too much".
I also had the citations in the Python documentation version.